charts/firecloudui/templates/deployment.yaml

```diff
@@ -0,0 +1,99 @@
+{{- $imageTag := .Values.app.image.tag | default .Values.global.applicationVersion -}}
+{{- $legacyResourcePrefix := .Values.legacyResourcePrefix | default .Values.name -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}-deployment
+  labels:
+{{ include "firecloudui.labels" . | indent 4 }}
```

Comment:
I know this is inconsistent in some of the older charts, but it would be good to be consistent about use  of `| indent x` vs `| nindent x`

Reply:
preference for the ` | nindent x` style.

Reply:
Good to know; the inconsistency in this PR is likely due to my buffet approach to grabbing what I needed. Will standardize on `nindent x`.